content/en/ecosystem/integrations.md

```diff
@@ -30,12 +31,12 @@ _Projects are listed alphabetically_
 
 The following open source projects use OpenTelemetry components.
 
-| External Project                                                                               | Applicable OpenTelemetry Components |
-| ----------------                                                                               | ----------------------------------- |
-| [Jaeger](https://www.jaegertracing.io/docs/1.21/opentelemetry/) [^not-ga]                      | Collector                           |
-| [Spring Sleuth](https://github.com/spring-cloud-incubator/spring-cloud-sleuth-otel/) [^not-ga] | Java                                |
+| External Project                                                                           | Applicable OpenTelemetry Components |
+| ------------------------------------------------------------------------------------------ | ----------------------------------- |
+| [Docker buildx](https://github.com/docker/buildx/blob/master/docs/guides/opentelemetry.md) | Go                                  |
+| [Jaeger](https://www.jaegertracing.io/docs/latest/opentelemetry/)                          | Collector                           |
```

Comment:
@svrnm, the Jaeger link is no longer valid:

```console
$ curl -I https://www.jaegertracing.io/docs/latest/opentelemetry/
HTTP/2 301 
...
location: /docs/1.42/opentelemetry/
...
$ curl -I https://www.jaegertracing.io/docs/1.42/opentelemetry/
HTTP/2 404 
```

From https://www.jaegertracing.io/docs/1.42/getting-started/:

> Historically, the Jaeger project supported its own SDKs (aka tracers, client libraries) that implemented the OpenTracing API. The [Client Libraries](https://www.jaegertracing.io/docs/1.42/client-libraries) section provides information about how to use the OpenTracing API and how to initialize and configure Jaeger tracers. As of 2022, the Jaeger SDKs are no longer supported, and all users are advised to migrate to OpenTelemetry.

I'll create an issue to track this but I wanted to flag it in context.